tests/h/presenters/organization_json_test.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import mock
+
+from h.presenters.organization_json import OrganizationJSONPresenter
+from h.resources import OrganizationResource
+
+
+class TestOrganizationJSONPresenter(object):
+    def test_organization_asdict_no_logo(self, factories, pyramid_request):
+        organization = factories.Organization(name='My Org', logo=None)
+        organization_resource = OrganizationResource(organization, pyramid_request)
+
+        presenter = OrganizationJSONPresenter(organization_resource)
+
+        assert presenter.asdict() == {
+            'name': 'My Org',
+            'id': organization.pubid,
+            'logo': None,
+        }
+
+    def test_organization_asdict_with_logo(self, factories, pyramid_request):
+        pyramid_request.route_url = mock.Mock()
```

Comment:
Yep! I've used that approach in other tests.

I mocked `request.route_url` here because it allowed me a more explicitly-readable way to express that the `logo` property should be the return value from `route_url`— it feels more like  a `mock` thing here than an actual routing thing.

Perhaps @seanh could chime in with his opinion on this one?